_posts/2024-02-09-release.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Release 2024.06
+author: Priya
+date: 2024-02-09
+category: Jekyll
+layout: post
+---
+### Enhancements:
+
+#### Relocate the Delete button for views within the preview panel:
+The 'Delete selected views' button for views on data libaray has been relocated within the preview panel to enhance user accessibility.
```

Comment:
The 'Delete selected views' button for views on data libaray has been relocated within the preview panel.